code/solutions/package-and-deploy/lambda/requirements.txt

```diff
@@ -0,0 +1 @@
+pytz==2019.3
```

Comment:
this is quite an old version, can we upgrade?

Reply:
Upgraded